File: temaMPPC#/temaMPPCSharp/temaMPPCSharp/repositoryes/RepoEmployee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using temaMPPCSharp.domains;

namespace temaMPPCSharp.repositoryes
{
    internal interface RepoEmployee
    {
        EmplyeeAtOffice login(String email, String password);

        EmplyeeAtOffice logout();
        EmplyeeAtOffice findOneByEmailAndPassword(String email, String password);
    }
}

File: an2/Medii de proiectare si programare/java/tema7/temaJavaMPP/ChatPersistence/src/main/java/com/example/repositoryes/interfaces/RepoEmployee.java
package com.example.repositoryes.interfaces;

import com.example.EmployeeAtOffice;

public interface RepoEmployee extends Repository<Integer, EmployeeAtOffice> {
    boolean logIn(String email, String password);

    void logOut();

    EmployeeAtOffice findOneByEmailAndPassword(String email, String password);
}
